<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MPVConfigWidget</class>
 <widget class="QWidget" name="MPVConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>431</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item>
    <widget class="QGroupBox" name="grpGeneral">
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,1">
      <item row="0" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
       <widget class="QCheckBox" name="kcfg_autoSyncEnabled">
        <property name="text">
         <string>A/V synchronization factor:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="kcfg_autoSyncFactor">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpVideo">
     <property name="title">
      <string>Video</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,0">
      <item row="1" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
       <widget class="QCheckBox" name="kcfg_hwDecodeEnabled">
        <property name="text">
         <string>Hardware video decoding:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="kcfg_hwDecode">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="kcfg_frameDropping">
        <property name="text">
         <string>Allow frame dropping</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="kcfg_renderMethod">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Native window embedding</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OpenGL render API</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
       <widget class="QLabel" name="label_renderMethod">
        <property name="text">
         <string>Video rendering method</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_renderMethod</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpAudio">
     <property name="title">
      <string>Audio</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,1">
      <item row="3" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
       <widget class="QCheckBox" name="kcfg_volumeAmplificationEnabled">
        <property name="text">
         <string>Volume amplification:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
       <widget class="QCheckBox" name="kcfg_audioChannelsEnabled">
        <property name="text">
         <string>Audio channels:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
       <widget class="QCheckBox" name="kcfg_audioOutputEnabled">
        <property name="text">
         <string>Audio output driver:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QSlider" name="kcfg_volumeAmplification">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>100</number>
           </property>
           <property name="maximum">
            <number>600</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="pageStep">
            <number>50</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lab_volumeAmplification"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QSlider" name="kcfg_audioChannels">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>6</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lab_audioChannels"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="kcfg_audioOutput">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="kcfg_volumeNormalization">
        <property name="text">
         <string>Volume normalization</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>kcfg_autoSyncEnabled</tabstop>
  <tabstop>kcfg_autoSyncFactor</tabstop>
  <tabstop>kcfg_renderMethod</tabstop>
  <tabstop>kcfg_hwDecodeEnabled</tabstop>
  <tabstop>kcfg_hwDecode</tabstop>
  <tabstop>kcfg_frameDropping</tabstop>
  <tabstop>kcfg_audioOutputEnabled</tabstop>
  <tabstop>kcfg_audioOutput</tabstop>
  <tabstop>kcfg_audioChannelsEnabled</tabstop>
  <tabstop>kcfg_audioChannels</tabstop>
  <tabstop>kcfg_volumeAmplificationEnabled</tabstop>
  <tabstop>kcfg_volumeAmplification</tabstop>
  <tabstop>kcfg_volumeNormalization</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>kcfg_hwDecodeEnabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_hwDecode</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_audioOutputEnabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_audioOutput</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_audioChannelsEnabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_audioChannels</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_volumeAmplificationEnabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_volumeAmplification</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_autoSyncEnabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_autoSyncFactor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_audioChannels</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lab_audioChannels</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_volumeAmplification</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lab_volumeAmplification</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
